agents/blueprints/profile_extractor_agent.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
This file is not properly formatted, is it possible to create simple pipeline to perform formatting and linting?